### DISCUSSION TITLE
Switching from Simple Mobiles to Fossify

### DISCUSSION THREAD

**Discussion by WebworkrNet** [FossifyOrg/General-Discussion#496] [Q&A] [answered]
I have already installed the Fossify app and set the new app as default. The new messages are already being processed via Fossify.

At first glance it looks like all messages have been transferred from the old app.

The messages are stored centrally by Android, right? Any app that processes SMS/MMS ultimately "just" accesses this storage, right? As a result, I can uninstall the old app without export/import without any risk, so I don't have to worry about losing any data, correct?

The Android data backup (currently via Google) remains unaffected?

Thanks

**Comment by Aga-C** [comment, marked as answer]
Yes.

**Reply by naveensingh** [reply]
Usually.

**Reply by WebworkrNet** [reply]
Thank you, I uninstalled the old app.
